feeds/svc.metrotransit.org.dmfr.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://dmfr.transit.land/json-schema/dmfr.schema-v0.3.0.json",
+  "feeds": [
+    {
+      "spec": "gtfs",
+      "id": "f-9zv-twin~cities~minnesota",
+      "urls": {
+        "static_current": "https://svc.metrotransit.org/mtgtfs/gtfs.zip",
+        "static_planned": "http://svc.metrotransit.org/mtgtfs/next/gtfs.zip"
```

This is where the JSON Schema validation is currently failing:

```suggestion
        "static_planned": ["http://svc.metrotransit.org/mtgtfs/next/gtfs.zip"]
```

After this change, I think we should be ready to merge. Thanks.

---

Is there a place I can see the validation errors?

---

We don't yet run JSON Schema checks on GitHub Actions, but here's one way to run them locally:

```
wget https://dmfr.transit.land/json-schema/dmfr.schema-v0.3.0.json
npx ajv-cli test -s dmfr.schema-v0.3.0.json -d svc.metrotransit.org.dmfr.json --valid
```